specs/PushNotifications/ToastNotifications-spec.md

```diff
@@ -0,0 +1,1026 @@
+# Toast Notifications in Windows App SDK
+
+# Background
+
+Toast Notifications are UI popups that are added that contain rich text, controls and images to
+display a message to the user. They are developer driven and originate from a target application
+that is installed by a user on the local device. It is not sufficient for a toast popup on the
+screen to simply be displayed to the user, it also needs to be actionable. For example, the user
+should be able to click on a toast popup to launch an app in the correct context. For example, a
+news article related toast is expected to launch the News app along with the article in question in
+the foreground. Another actionable scenario is for the user to actually interact with the contents
+of the toast popup like a UI control button. For example, a messaging app like Teams that displays a
+message from another user could have a "Respond To" textbox and button so that the user can directly
+respond to the message in the popup without having to launch the application. This scenario triggers
+a background process on behalf of the application (with no UI) which inturn processes the "Reply To"
+message and forwards the response to the other device.
+
+Toast messages could have a local or cloud based origin. In the case of local toasts, the message
+always originates from the app installed on the device. For cloud toasts, the message always
+originates from a remote application service that targets the locally installed app in question.
+
+Here is a visual representation of a Simple Toast with no interactive controls:
+
+![Toast Simple](https://docs.microsoft.com/en-us/windows/apps/design/shell/tiles-and-notifications/images/send-toast-02.png)
```

Comment:
don't use /en-us links - remove that part of the url so that the link opens in the user's language.

(here and below)

Reply:
Thanks for pointing out. Fixed the links.